src/OceanSeaIceModels/PrescribedAtmospheres.jl

```diff
@@ -320,32 +321,37 @@ function Base.show(io::IO, pa::PrescribedAtmosphere)
 end
 
 function default_atmosphere_velocities(grid, times)
-    ua = FieldTimeSeries{Center, Center, Nothing}(grid, times)
-    va = FieldTimeSeries{Center, Center, Nothing}(grid, times)
+    bcs = FieldBoundaryConditions(grid, (Center, Center, Nothing))
+    ua  = FieldTimeSeries{Center, Center, Nothing}(grid, times; boundary_conditions=bcs)
+    va  = FieldTimeSeries{Center, Center, Nothing}(grid, times; boundary_conditions=bcs)
     return (u=ua, v=va)
 end
 
 function default_atmosphere_tracers(grid, times)
-    Ta = FieldTimeSeries{Center, Center, Nothing}(grid, times)
-    qa = FieldTimeSeries{Center, Center, Nothing}(grid, times)
+    bcs = FieldBoundaryConditions(grid, (Center, Center, Nothing))
+    Ta  = FieldTimeSeries{Center, Center, Nothing}(grid, times; boundary_conditions=bcs)
+    qa  = FieldTimeSeries{Center, Center, Nothing}(grid, times; boundary_conditions=bcs)
     parent(Ta) .= 273.15 + 20
     return (T=Ta, q=qa)
 end
 
 function default_downwelling_radiation(grid, times)
-    Qℓ = FieldTimeSeries{Center, Center, Nothing}(grid, times)
-    Qs = FieldTimeSeries{Center, Center, Nothing}(grid, times)
+    bcs = FieldBoundaryConditions(grid, (Center, Center, Nothing))
+    Qℓ  = FieldTimeSeries{Center, Center, Nothing}(grid, times; boundary_conditions=bcs)
+    Qs  = FieldTimeSeries{Center, Center, Nothing}(grid, times; boundary_conditions=bcs)
     return TwoBandDownwellingRadiation(shortwave=Qs, longwave=Qℓ)
 end
 
 function default_freshwater_flux(grid, times)
-    rain = FieldTimeSeries{Center, Center, Nothing}(grid, times)
-    snow = FieldTimeSeries{Center, Center, Nothing}(grid, times)
+    bcs  = FieldBoundaryConditions(grid, (Center, Center, Nothing))
+    rain = FieldTimeSeries{Center, Center, Nothing}(grid, times; boundary_conditions=bcs)
+    snow = FieldTimeSeries{Center, Center, Nothing}(grid, times; boundary_conditions=bcs)
     return (; rain, snow)
 end
 
 function default_atmosphere_pressure(grid, times)
-    pa = FieldTimeSeries{Center, Center, Nothing}(grid, times)
+    bcs = FieldBoundaryConditions(grid, (Center, Center, Nothing))
+    pa  = FieldTimeSeries{Center, Center, Nothing}(grid, times; boundary_conditions=bcs)
```

Comment:
Why don't these boundary conditions get added already? This seems like a change that belongs in Oceananigans. What's the motivation for doing this here? You will have to do this every time you make a FieldTimeSeries right, isn't it a bit much?

Reply:
Should we change the default here?

https://github.com/CliMA/Oceananigans.jl/blob/a93dcb28fcca69b3f7867a0c3fc07dbf88136df5/src/OutputReaders/field_time_series.jl#L376

Reply:
I guess even if we do, we'd probably don't add defaults that have a Nothing z-location

Reply:
@simone-silvestri just for clarification: is this only needed for FieldTimeSeries that are restricted on the surface? Or you are saying that we need to include boundary conditions in the constructors of FieldTimeSeries when we plan to interpolate those time series; this happens to be one example of such case you stumbled upon?

Reply:
i'll do it